aten/src/TH/generic/THTensorMoreMath.cpp

```diff
@@ -484,7 +484,8 @@ void THTensor_(cross)(THTensor *r_, THTensor *a, THTensor *b, int dimension)
     }
   }
 
-  if(dimension < 0)
+  int64_t dimension = -1;
```

Comment:
" if(THTensor_(nDimensionLegacyNoScalars)(a) != " you don't actually need all this, because a 0-dimensional tensor can't have a dimension of size 3.

Reply:
Removed TH implementation of cross